frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/obc-test.spec.ts

```diff
@@ -0,0 +1,165 @@
+    cy.get('@secrets').should(($el) => {
+      const childCount = $el.length;
+      const hiddenText = 'Value hidden';
+      const elText = $el.text();
+      const secrets = elText.split(hiddenText).slice(1);
+      secrets.forEach((arrEl: string) => {
+        expect(arrEl).to.equal(MASK);
+      });
+      expect(secrets.length).to.equal(childCount);
+    });
+  });
+
+  it('Test if secret data can be revealed', () => {
+    obcHandler.revealHiddenValues();
+    cy.byTestID('co-copy-to-clipboard__text').then(($el) => {
+      cy.byTestID('secret-data__dt').should(($h) => {
+        expect($h[0].innerText).to.equal('Endpoint');
+        expect($el[0].innerText).to.include(NS);
+        expect($h[2].innerText).to.equal('Access Key');
+        expect($el[2].innerText).to.match(new RegExp(ACCESS_KEY));
+        expect($h[3].innerText).to.equal('Secret Key');
+        expect($el[3].innerText).to.match(new RegExp(SECRET_KEY));
+      });
+    });
+  });
+
+  it('Test if secret data can be hidden again', () => {
+    obcHandler.hideValues();
+    cy.byTestID('co-copy-to-clipboard__text').as('secrets');
+    cy.get('@secrets').should(($el) => {
+      const childCount = $el.length;
+      const hiddenText = 'Value hidden';
+      const elText = $el.text();
+      const secrets = elText.split(hiddenText).slice(1);
+      secrets.forEach((arrEl) => {
+        expect(arrEl).to.equal(MASK);
+      });
+      expect(secrets.length).to.equal(childCount);
+    });
+  });
+
+  it('Test if labels and annotations are shown correctly', () => {
+    cy.byTestID('label-list').contains(NOOBAA_LABEL);
+    cy.byTestSelector('details-item-value__Annotations').contains(NO_ANNOTATIONS);
+  });
+
+  it('Test if namespace and secret are shown correctly', () => {
+    obcHandler.assertNamespaceExists();
+    cy.get('[title="Secret"]')
+      .siblings()
+      .contains(OBC_NAME);
+  });
+
+  it('Test if status and storage class are shown correctly', () => {
+    cy.byTestID('status-text').contains(BOUND);
+    cy.byLegacyTestID('openshift-storage.noobaa.io').contains(OBC_STORAGE_CLASS_EXACT);
+  });
+
+  it('Test if Object Bucket is created', () => {
+    const obName = `obc-${testName}-${OBC_NAME}`;
+    cy.contains(obName).click();
+    cy.byTestID('co-m-loader').should('not.exist');
+    cy.byLegacyTestID('resource-title').as('title');
+    cy.get('@title')
+      .should('be.visible')
+      .then(($el) => {
+        expect($el.text()).to.equal(obName);
+      });
+    cy.byTestID('resource-status').as('status');
+    cy.get('@status')
+      .should('be.visible')
+      .then(($el) => {
+        expect($el.text()).to.equal(BOUND);
+      });
+  });
+
+  it('Test attachment to a Deployment', () => {
+    cy.clickNavLink(['Storage', 'Object Bucket Claims']);
+    cy.get('tr')
+      .contains(OBC_NAME)
+      .should('be.visible')
+      .byLegacyTestID('kebab-button')
+      .first()
+      .click();
+    cy.contains(ATTACH_TO_DEPLOYMENT).click();
```

Comment:
```suggestion
   listPage.rows.shouldBeLoaded();
   listPage.rows.clickKebabAction(OBC_NAME, ATTACH_TO_DEPLOYMENT);
```
if `ATTACH_TO_DEPLOYMENT` is only used here then directly use it here without declaring it as a constant. 
Import `listPage` from `integration-tests-cypress/views/list-page`